### ISSUE TITLE
`eris init --dir`

### ISSUE THREAD

**Issue by compleatang** [eris-ltd/eris-cli#377]
(mainly for testing purposes) we should be able to init into a directory other than ~/.eris.

**Comment by zramsay** [comment]
closing in favour of task for #309

**Comment by zramsay** [comment]
How important is this? Because the init sequence is tied to /common and thus `ResolveErisRoot()`, it all has to be unbundled (a bunch of ~unnecessary code). If this is only for tests, we already have `config.ChangeErisDir(erisDir)` with `var erisDir = path.Join(os.TempDir(), "eris")` as part of the `TestsInit` sequence in `testutils` which leaves your `~/.eris` alone

**Comment by compleatang** [comment]
This is low priority. We may want to refactor `ResolveErisRoot()`, but we dont' have to do it today. 

The problem is that during the testing sequence we first `eris init --yes` so that we can start IPFS which is needed to be running for the first tranche of package tests (I think). If we reworked, predominantly `files_tests.go` so that it did not expect IPFS running then we could remove that `eris init --yes` in test_tool.sh which would stop messing with my local ~/.eris directory (which is what was happening when I filed the issue).
